### PR TITLE
s2417: https migration for canfar documents pages

### PR DESCRIPTION
**Pull request by hjeeves**

### CODE REVIEW

File: _config.yml [hunk 1]
```diff
@@ -5,6 +5,8 @@ relative_permalinks: false
 markdown: kramdown
 site_env: prod
 env:
+    hj:
+        hj: 'http://jeevesh.cadc.dao.nrc.gc.ca'
```
Comment (at88mph) [inline _config.yml]: Omit the `_config.yml` from the commits.

File: docs/quick_start_en.md [hunk 2]
```diff
@@ -203,7 +203,7 @@ vcp 1056213p.cat vos:[username]
 
 </div>
 
-Verify that the file is properly uploaded by pointing your browser to the [VOSpace browser interface](http://www.canfar.phys.uvic.ca/vosui).
+Verify that the file is properly uploaded by pointing your browser to the [VOSpace browser interface](https://www.canfar.phys.uvic.ca/vosui).
```
Comment (at88mph) [inline docs/quick_start_en.md]: This should be https://www.canfar.net/storage/list/

File: docs/quick_start_fr.md [hunk 3]
```diff
@@ -7,7 +7,7 @@ permalink: /fr/docs/commencer/
 ---
 
 
-Before starting this example, you will need to [register](http://apps.canfar.net/canfar/login.html) to CANFAR. The CANFAR team will take care of your registration to Compute Canada infrastructure.
+Before starting this example, you will need to [register](https://www.canfar.net/canfar/login.html) to CANFAR. The CANFAR team will take care of your registration to Compute Canada infrastructure.
```
Comment (at88mph) [inline docs/quick_start_fr.md]: The `register` link is actually https://www.canfar.net/en/register/.

File: docs/storage_en.md [hunk 4]
```diff
@@ -12,9 +12,9 @@ VOSpace is the CANFAR storage system , an implementation of	the [Virtual Observa
 It is intended to be used for storing the output of the CANFAR processing system and also for sharing files between members of a collaboration.
 If the data you want to process is not already on a CADC archive, you can stage it on a VOSpace for further processing. Files in VOSpace are mirrored in four physical locations, so they are secure against disk failure.
 
-You will need to [register](http://apps.canfar.net/canfar/auth/request.html).
+You will need to [register](https://www.canfar.net/canfar/auth/request.html).
```
Comment (at88mph) [inline docs/storage_en.md]: The register link is actually https://www.canfar.net/en/register/.

File: resources/docs/quick_start_en.md [hunk 5]
```diff
@@ -198,7 +198,7 @@ $ vcp 1056213p.cat vos:[username]
 
 </div>
 
-Verify that the file is properly uploaded by pointing your browser to the [VOSpace browser interface](http://www.canfar.phys.uvic.ca/vosui).
+Verify that the file is properly uploaded by pointing your browser to the [VOSpace browser interface](https://www.canfar.phys.uvic.ca/vosui).
```
Comment (at88mph) [inline resources/docs/quick_start_en.md]: Storage UI is at https://www.canfar.net/storage/list/

File: resources/docs/quick_start_fr.md [hunk 6]
```diff
@@ -198,7 +198,7 @@ $ vcp 1056213p.cat vos:[username]
 
 </div>
 
-Verify that the file is properly uploaded by pointing your browser to the [VOSpace browser interface](http://www.canfar.phys.uvic.ca/vosui).
+Verify that the file is properly uploaded by pointing your browser to the [VOSpace browser interface](https://www.canfar.phys.uvic.ca/vosui).
```
Comment (at88mph) [inline resources/docs/quick_start_fr.md]: Storage UI is at https://www.canfar.net/storage/list/

**Review by at88mph — CHANGES_REQUESTED** [review]

File: _config.yml (hunk 1 above)
Comment (hjeeves) [reply _config.yml]: ok

**Review by at88mph — APPROVED** [review]

**Pull request merged**